db/index.js

```diff
@@ -3,26 +3,46 @@
 const mysql = require('mysql');
 const dbConfig = require('./config.js');
 
-const connection = mysql.createConnection(dbConfig);
-connection.connect((err) => {
-  if (err) {
-    console.log(err);
-  } else {
-    console.log('CONNECTED TO DB!');
-  }
-});
+const dbConnection = mysql.createConnection(dbConfig);
+dbConnection.connect();
```

Comment:
have you tried running jest with this .connect() line out? I didn't need it in mine but maybe you have a different set up.

Reply:
Thanks for pointing out! Just tried, and it seem to be working both ways! Will merge and remove it in the next branch!